cmd/frontend/graphqlbackend/textsearch.go

```diff
@@ -788,6 +795,32 @@ func searchFilesInRepos(ctx context.Context, args *search.TextParameters) (res [
 	return flattened, common, nil
 }
 
+const maxUnindexedRepoRevSearchesPerQuery = 200
+
+// limitSearcherRepos imposes a limit on the number of repo@revs that would go
+// to the unindexed searcher codepath, because sending this many requests to
+// searcher would otherwise result in the system and network being flooded with
+// requests that merely result in timeouts and take a long time to complete.
+//
+// It returns the new repositories destined for the unindexed searcher code
+// path, and the repositories that are limited / excluded.
+//
+// The input list is not copied (a slice to it is returned).
```

Comment:
```suggestion
// A slice to the input list is returned, it is not copied.
```

IDK don't feel strongly about this.